src/main/java/net/sf/jabref/sql/DBConnectDialog.java

```diff
@@ -37,6 +37,11 @@
  */
 public class DBConnectDialog extends JDialog {
 
+    private static List<String> FORMATTED_NAMES = Arrays
```

Comment:
Make it non-static and move to the constructor since it is only used there as far as I can see?